tests/python-gpu/test_gpu_prediction.py

```diff
@@ -147,8 +147,10 @@ def predict_dense(x):
             copied_predt = cp.array(booster.predict(d))
             return cp.all(copied_predt == inplace_predt)
 
-        for i in range(10):
-            run_threaded_predict(X, rows, predict_dense)
+        # Don't do this on Windows, see issue #5793
+        if not sys.platform.startswith("win"):
+            for i in range(10):
+                run_threaded_predict(X, rows, predict_dense)
```

Comment:
Let's mark the test as skipped explicitly, so that we can remember to fix it:
```suggestion
        # Don't do this on Windows, see issue #5793
        if sys.platform.startswith("win"):
            pytest.skip('Multi-threaded in-place prediction with cuPy is not working on Windows')
        for i in range(10):
            run_threaded_predict(X, rows, predict_dense)
```